Overview.md

```diff
@@ -23,6 +23,21 @@ When the solution is just deployed, the monitoring worker does not have any acce
 Go to the deployed managed application and open the Parameters and Outputs blade. Copy the value of the monitoring Identity output.  
 < screenshot here >  
 Provide this identity with a Reader role to your subscription ([more info](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/howto-assign-access-portal))
+
+## Configure ADF Diagnostic Settings
+Configure diagnostic setting to send platform logs and metrics to Log Analytics Workspace.
+
+![Diagnostic Settings](Artifacts/DiagnosticSettings.PNG)
+1. Go to the menu of the created Data Factory.

```

Comment:
1. Not "created" but "Data Factory whic you are going to monitor".

Reply:
edited